docs/source/install.rst

```diff
@@ -2,6 +2,28 @@
 Install
 =======
 
-In Python 3.6 or later, Compose can be installed by running the following commmand:
+Compose is available on PyPI and Conda-forge for Python version 3.6 or later.
+
+:code:`pip`
```

Comment:
I find the pink code formatting a little harsh / jarring - can we use the regular heading formatting for the pip and conda sections?